test/Libraries/Microsoft.Extensions.AI.Tests/ChatCompletion/FunctionInvokingChatClientTests.cs

```diff
@@ -503,7 +503,7 @@ async Task InvokeAsync(Func<Task> work)
 
                 for (int i = 0; i < activities.Count - 1; i++)
                 {
-                    // Activities are exported in the order of completion, so all except the last are children of the last (i.e., outer)
+                    // Activities are exported in the order of response, so all except the last are children of the last (i.e., outer)
```

Comment:
```suggestion
                    // Activities are exported in the order of completion, so all except the last are children of the last (i.e., outer)
```

Arguably the new phrasing would be valid too, but I think the older one is clearer